salt/modules/opkg.py

```diff
@@ -56,9 +56,11 @@ def __virtual__():
     '''
     Confirm this module is on a nilrt based system
     '''
-    if __grains__.get('os_family', False) == 'NILinuxRT':
+    try:
+        os.listdir(OPKG_CONFDIR)
```

Comment:
Why would you `os.listdir` it only in order to know that the directory is there? This generates lots of stats for literally nothing. Why not just:

```python
def __virtual__():
    return not os.path.isdir(OPKG_CONFDIR) and (False, "Not OpenEmbedded system") or True

Reply:
Ah. Forgot about that method. Will do.